pkg/linode-bs/controllerserver.go

```diff
@@ -102,6 +102,8 @@ func (linodeCS *LinodeControllerServer) CreateVolume(ctx context.Context, req *c
 		volumeContext[LuksKeySizeAttribute] = req.Parameters[LuksKeySizeAttribute]
 	}
 
+	tags := req.Parameters[VolumeTags]
```

Comment:
It is weird that VolumeTags comes from the LUKS constants

Reply:
req.Paramaters here is the params on the storage class

Reply:
ah the const is defined in luks to keep things in one place - can move it here if it makes more sense

Reply:
I'd move the few non LUKS constants to here and keep the LUKS stuff in LUKS. The drivername and the volume tags and phases I think should be in this file.

Reply:
moved it over